src/BenchmarkDotNet/Validators/DotNetSdkValidator.cs

```diff
@@ -32,7 +32,11 @@ public static IEnumerable<ValidationError> ValidateCoreSdks(string? customDotNet
 
         public static IEnumerable<ValidationError> ValidateFrameworkSdks(BenchmarkCase benchmark)
         {
-            var requiredSdkVersion = benchmark.GetRuntime().RuntimeMoniker.GetRuntimeVersion();
+            var targetRuntime = benchmark.Job.Environment.HasValue(EnvironmentMode.RuntimeCharacteristic)
+                ? benchmark.Job.Environment.Runtime
+                : ClrRuntime.GetTargetOrCurrentVersion(benchmark.Descriptor.WorkloadMethod.DeclaringType.Assembly);
```

Comment:
What's the difference? It looks like `benchmark.GetRuntime()` does the exact same thing.

Reply:
Oh I see, the other code is calling `RuntimeInformation.GetTargetOrCurrentRuntime` and this changes it to `ClrRuntime.GetTargetOrCurrentVersion`.